jetty-http/src/main/java/org/eclipse/jetty/http/compression/HuffmanDecoder.java

```diff
@@ -51,17 +64,19 @@ public String decode(ByteBuffer buffer) throws EncodingException
             while (_bits >= 8)
             {
                 int c = (_current >>> (_bits - 8)) & 0xFF;
-                _node = tree[_node * 256 + c];
+                _node = Huffman.tree[_node * 256 + c];
                 if (rowbits[_node] != 0)
                 {
-                    if (rowsym[_node] == EOS)
+                    if (rowsym[_node] == Huffman.EOS)
                     {
                         reset();
                         throw new EncodingException("eos_in_content");
                     }
 
                     // terminal node
-                    _utf8.append((byte)(0xFF & rowsym[_node]));
+                    int i = 0xFF & rowsym[_node];
+                    i = HttpTokens.sanitizeFieldVchar(i);
+                    _builder.append((byte)i);
```

Comment:
So this is kind of double sanitizing now.  The 0xFF is already cutting off big values.   The `rowsym` table is already characters, so why not leave it as a char then sanitize using the char method?